pkg/odo/cli/service/create.go

```diff
@@ -31,10 +33,12 @@ A full list of service types that can be deployed are available using: 'odo cata
 
 // ServiceCreateOptions encapsulates the options for the odo service create command
 type ServiceCreateOptions struct {
-	parameters  []string
-	plan        string
-	serviceType string
-	serviceName string
+	parameters    []string
+	plan          string
+	serviceType   string
+	serviceName   string
+	parametersMap map[string]string
```

Comment:
what is a difference between `parametersMap` and `parameters`? could we have just one?

Reply:
Basically, `parameters` holds the parameters provided via flag values while `parametersMap` is populated from the interactive mode and the provided parameters as well as the function that does the actual work expects such a map. `parameters` is definitely needed and having `parametersMap` make things smoother.

Reply:
can you please add this as a code comment?

Reply:
hmm, weird, I didn't see these comments in the global conversation… only saw them from the emails. Annoying! :(

Reply:
Fixed.